custom_components/alexa_media/__init__.py

```
@@ -3235,7 +3236,8 @@ async def test_login_status(hass, config_entry, login) -> bool:
         hass,
         title="Alexa Media Reauthentication Required",
         message=message,
-        notification_id=f"alexa_media_{slugify(login.email)}{slugify(login.url[7:])}",
+        host = urlparse(login.url).hostname or login.url
```

Pylint (4.0.5)

[error] Parsing failed: 'invalid syntax. Perhaps you forgot a comma? (alexa_media.__init__, line 3239)'

(E0001)